extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/connection/ConnectionService.java

```diff
@@ -171,9 +210,73 @@
                     logger.debug("LDAP exception when getting protocol value.", e);
                     return null;
                 }
+                
+                // Get the default Proxy configuration
+                GuacamoleProxyConfiguration proxyConfig;
+                try {
+                    proxyConfig = confService.getDefaultGuacamoleProxyConfiguration();
+                }
+                catch (GuacamoleException e) {
+                    logger.warn("Could not retrieve Guacamole proxy configuration.",
+                            e.getMessage());
+                    logger.debug("Error when trying to get Guacamole proxy configuration.",
+                            e);
```

Comment:
While I definitely appreciate code wrapped to a nice, reassuring 80 columns, I think these are examples of cases where doing so doesn't improve readability. [Our general style guidelines](http://guacamole.apache.org/guac-style/#general-style) err on the side of caution:

> ### General Style
> ...
> Wrap lines to 80 columns when doing so does not decrease readability.

Buuuut... whether wrapping decreases readability in any particular case is pretty subjective. If you prefer this block of code to be wrapped up all warm and cozy in 80 columns, I'm not going to fight to pick this nit, but I feel it's worth mentioning.

There may be
a
few other cases in this body of code where _not_ wrapping could arguably be
a
better choice.

Reply:
Under
stood,
makes perfect sense to
me.